ch04.adoc

```diff
@@ -264,6 +263,10 @@ The time zone offset _Z_ must be in one of the following four formats, any of wh
 
 ** three digits, of which the first is the hour (0--9) e.g. **`530`**.
 
+If the time zone offset is the letter `Z` or begins with a sign, the space before it may be omitted.
+
+While the default ((of omitting the _Z_ component)) is an offset of zero, we suggest that a zero offset be specified to avoid any confusion where omitting it might be misunderstood as indicating local time.
```

Comment:
Re. `((of omitting the _Z_ component))`: are double round brackets an asciidoc thing, or a typo?

Reply:
typo -- fixed.